README.md

```diff
@@ -57,12 +57,14 @@ Quit `psql` using the following command:
 \q
 ```
 
-On Linux, you will also need to create a Linux system user with a name matching the user name you used in the database. It will prompt you to create a password for the user - choose the same password as for the database above.
+On Linux, you will also need to create a Linux system user with a name matching the user name you used in the database. Generate a random password for the user:
 
 ```bash
-sudo adduser <user name>
+openssl rand -hex 16
```

Comment:
this is wrong, it removes the `adduser` command

Reply:
Yes, forgot to copy a part 
[0b7e8e9](https://github.com/upleveled/next-js-example-spring-2024-atvie/pull/6/commits/0b7e8e9f9d8c419ba0ff15ca96bd47d2e91c9162)